tree_for_register_to_template.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+usage() { echo "$(basename $0) [-t tractoflow/results] [-r rbx/results_rbx/] [-n noddi/results] [-f freewater_flow/results] [-o output]" 1>&2; exit 1; }
+
+while getopts "t:o:" args; do
+    case "${args}" in
+        t) t=${OPTARG};;
+        r) r=${OPTARG};;
```

Comment:
remove r) line

Reply:
This will register the output bundles from rbx into template space.